tests/testCadenceMetrics.py

```diff
@@ -153,13 +153,13 @@ def testNVisitsPerNightMetric(self):
         # All nights have one visit.
         expected_result = np.zeros(len(bins) - 1, dtype=int)
         expected_result[1] = len(data)
-        self.assertTrue(np.all(result == expected_result))
+        np.testing.assert_array_equal(result, expected_result)
```

Comment:
I actually think that the self.assertTrue(np.all()) is pretty equivalent to the numpy.testing.assert_array_equal, isn't it? It's fine to change it, I just was thinking that the more basic assert was a better way to go when I wrote this and if it's not, I'd like to change my approach.

Reply:
I believe the philosophical justification of using `np.testing.assert_array_equal()` is that, when it fails, it prints both arrays (or, at least, as much of them as will fit) to stdout so that you can get a sense of why the test failed.  `self.assertTrue()` just prints `False is not True` when it fails.

Reply:
Okay, thanks. That's a reasonable, good, reason.